src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenSystemNullableTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
+{
+    public class CodeGenSystemNullableTests : CSharpTestBase
```

Comment:
>CodeGenSystemNullableTests [](http://example.com/codeflow?start=17&length=26)

It doesn't feel like we need a new test file just for this. Can you add the tests into `src\Compilers\CSharp\Test\Emit2\Semantics\PatternMatchingTests5.cs` instead?